modules/reloadBidAdapter.js

```diff
@@ -0,0 +1,513 @@
+import {
+  BANNER
+}
+from 'src/mediaTypes';
+import {
+  registerBidder
+}
+from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'reload';
+
+const VERSION_ADAPTER = "1.0";
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: [],
```

Comment:
Please remove aliases property if you are not creating any alias

Reply:
Fixed